skilloracle/skilloracle/skilloracle/skilloracle.py

```diff
@@ -0,0 +1,193 @@
+import time
```

Comment:
This deep of a hierarchy with every level named 'skilloracle' is confusing. At the very least, if this .py file is the only thing that can be imported from this level, we can probably just put the contents in the __init__.py file. But I think there's other stuff to unpack here. What are all the levels?

Reply:
Agreed, I'll move things around to reflect something more like this (as discussed in #mlops):

```
Skill-labeler/skilloracle
Skill-labeler/test #all unit tests for each microservice
Skill-labeler/skilloracle/api # see next comment
Skill-labeler/skilloracle/endpoint # micro service related
Skill-labeler/skilloracle/oracle # ml smarts
Skill-labeler/skilloracle/webserver.py, run.sh # stands up service, its API
```